docs/framework/net-native/runtime-directive-policy-settings.md

```diff
@@ -57,24 +57,25 @@ Runtime directives files recognize three categories of policy types: reflection,
-  |[\<AttributeImplies>](attributeimplies-element-net-native.md)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|
-  |[\<Event>](event-element-net-native.md)|||||
-  |[\<Field>](field-element-net-native.md)|![check mark icon](../../media/check-mark.png)||||
-  |[\<GenericParameter>](genericparameter-element-net-native.md)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|
-  |[\<ImpliesType>](impliestype-element-net-native.md)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|
-  |[\<Method>](method-element-net-native.md)|||||
-  |[\<MethodInstantiation>](methodinstantiation-element-net-native.md)|||||
-  |[\<Namespace>](namespace-element-net-native.md)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|
-  |[\<Parameter>](parameter-element-net-native.md)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|
-  |[\<Property>](property-element-net-native.md)|![check mark icon](../../media/check-mark.png)||||
-  |[\<Subtypes>](subtypes-element-net-native.md)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|
-  |[\<Type>](type-element-net-native.md)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|
-  |[\<TypeInstantiation>](typeinstantiation-element-net-native.md)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|
-  |[\<TypeParameter>](typeparameter-element-net-native.md)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|
+  > [!div class="mx-tdBreakAll"]
+  > |Element|Serialize|DataContractSerializer|DataContractJsonSerializer|XmlSerializer|
+  > |-|:-:|:-:|:-:|
+  > |[\<Application>](application-element-net-native.md)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|![check mark icon](../../media/check-mark.png)|
```

Comment:
The icons need to have an alt text added to them to solve the accessibility issue.

Reply:
They do - it is "check mark icon". See https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=master#standard-conceptual-images-default-markdown.

Reply:
Oh doh, I was just thinking it still had to be a sentence after the path. Forgive me! 😄